src/calculateRentalCost.js

```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  if (days >= 7) {
```

Comment:
Replace the magic numbers (40, 50, and 20) with named constants for better readability and maintainability.